src/applications/disability-benefits/2346/schemas/address-schema.js

```diff
@@ -361,6 +361,19 @@ export const addressUISchema = (
         hideIf: () => !isMilitaryBaseAddress,
         hideOnReviewIfFalse: true,
         useDlWrap: true,
+        updateSchema: (formData, schema, uiSchema) => {
+          const countryFormData = get(path, formData);
+          const countryNameNormalized = countryFormData.toUpperCase();
+          if (
+            countryNameNormalized.includes('ARMED FORCES') ||
+            countryNameNormalized.includes('AF,EU,ME,CA')
```

Comment:
These should all be individual.  I don't think they would pass us a string of 4 countries as a single country.

Reply:
or better yet, we have a list of acceptable countries coming from schema somewhere.. you can just say something like 
```
country && listOfGoodCountries.includes(country)
````
instead of doing the inverse - this would end up getting you all the inclusion you need without having to do multiple inclusion checks for a case we are trying to avoid